packages/contracts-bedrock/slither-report.json

```diff
@@ -0,0 +1,1274 @@
+[
```

Comment:
This file was deleted previously, looks like it was rebased back in ?

Reply:
Deleted again